contracts/disputes/IDisputeManager.sol

```diff
@@ -12,6 +12,13 @@ interface IDisputeManager {
         QueryDispute
     }
 
+    enum DisputeStatus {
+        Accepted,
```

Comment:
This will make non-existent disputes show up as Accepted, because it will be the default status (0). I'd suggest adding a `Null` status or similar as first value for the enum.

Reply:
Ok cool sounds like a plan! I'll modify those here in a bit. I'll also reach out on discord.